scripts/e2e/kafka.sh

```diff
@@ -129,4 +129,4 @@ main() {
   success="true"
 }
 
-main "$@"
+main "$@"
```

Comment:
these last two files have no changes. please `git checkout main $file` to restore them